Solutions/Microsoft 365 Defender/Hunting Queries/Delivered Bad Emails from Top bad IPv4 addresses .yaml

```diff
@@ -0,0 +1,17 @@
+id: cdac93ef-56c0-45bf-9e7f-9cbf0ad06567
+name: Delivered Bad Emails from Top bad IPv4 addresses  
+description: |
+  This query determines Bad Emails from Top bad IPv4 addresses .
+requiredDataConnectors:
+- connectorId: OfficeATP
+  dataTypes:
+  - EmailEvents
+tactics:
+- Initial access
+query: |
+  EmailEvents
+  | where ThreatTypes has "Malware" or ThreatTypes has "Phish" 
+  | summarize count() by SenderIPv4 
+  | where count_ > 5 // Arbitrary cutoff, increase or decrease as needed 
```

Comment:
Can we include this in the description that a hardcoded threshold is present which can be modified. Also, use a let statement to state this value.

Reply:
done